src/python/grpcio/grpc/__init__.py

```diff
@@ -424,6 +424,23 @@ def __init__(self, credentials):
         self._credentials = credentials
 
 
+class ServerCertificateConfig(object):
+    """A certificate config for use with an SSL-enabled Server, e.g., can
+    be returned in the certificate config fetching callback.
+
+    This class has no supported interface -- it exists to define the
+    type of its instances and its instances exist to be passed to
+    other functions.
+
+    NOTE: do not re-use instances of this class, i.e, always create a
```

Comment:
this is because C-core takes ownership this object's `grpc_ssl_server_certificate_config` member and so it then becomes un(re)usable.
alternatively, if we want to support re-use of `ServerCertificateConfig` instances --- not clear if that's useful --- then in cython we could make a dup copy of the `grpc_ssl_server_certificate_config` object (manually creating a second object, or perhaps adding in C-core a `grpc_ssl_server_certificate_config_dup()` API).